support.js

```diff
@@ -66,13 +66,8 @@ const registerHooks = () => {
     windowCoverageObjects = []
 
     const saveCoverageObject = (win) => {
-      // if the application code has been instrumented, then the app iframe "win.__coverage__" will be available,
-      // in addition, accessing win.__coverage__ can throw when testing cross-origin code,
-      // because we don't control the cross-origin code, we can safely return
-      let applicationSourceCoverage
-      try {
-        applicationSourceCoverage = win?.__coverage__
-      } catch {}
+      // if application code has been instrumented, the app iframe "window" has an object
+      const applicationSourceCoverage = win.__coverage__
```

Comment:
```suggestion
      // if the application code has been instrumented, then the app iframe "win.__coverage__" will be available,
      // in addition, accessing win.__coverage__ can throw when testing cross-origin code,
      // because we don't control the cross-origin code, we can safely return
      let applicationSourceCoverage
      try {
        applicationSourceCoverage = win?.__coverage__
      } catch {}
```